firestore-translate-text/functions/src/config.ts

```diff
@@ -15,7 +15,7 @@
  */
 
 export default {
-  languages: process.env.LANGUAGES.split(","),
+  languages: new Set(process.env.LANGUAGES.split(",")),
```

Comment:
To save duplication, it may be better to do:

```
languages: Array.from(
  new Set(process.env.LANGUAGES.split(","))
),
``

That way, `config.languages` is always an array throughout the extension rather than a Set.